.github/workflows/JirabotMerge.yml

```diff
@@ -182,6 +191,11 @@ jobs:
             jirabot_user = os.environ['JIRABOT_USERNAME']
             jirabot_pass = os.environ['JIRABOT_PASSWORD']
             jira_url = os.environ['JIRA_URL']
+
+            if not jira_url:
```

Comment:
This is fine, but the hardcoded url will need to be updated soon, and it feels like a workaround rather than a fix (if os.environ['JIRA_URL'] is still not working)

Reply:
Agreed, I wanted to get some more debugging information in place, but decided it would be worthwhile to have the backup as well.